docs/CONTRIBUTING.md

```diff
@@ -38,23 +38,32 @@ The project has the following file structure:
 - [/docs](../docs): Documentation about the repository.
 - [/modules](../modules): Customisable modules to create Google Cloud Platform infrastructure & supporting scripts.
 - [/radlab-launcher](../radlab-launcher): RAD-Lab Launcher and associated scripts to support the modules.
+- [/radlab-ui](../radlab-ui/): Source code to deploy RAD Lab UI.
 
-Every individual module is contained in it subdirectory and should follow these Terraform guidelines:
+Every individual module is contained in it subdirectory and should follow these guidelines:
 
 - Create at least a `versions.tf`, `main.tf`, `variables.tf` and `outputs.tf`.  It's ok to create additional files, to split resources between files and give them more meaningful names. 
 
-NOTE: Make sure all the variables are in alphabetical order in `variables.tf`
+    NOTE: Make sure all the variables are in alphabetical order in `variables.tf`
 
-- If any additional scripts (apart from **terraform** configs) are required to support the module either to *build* or *use* the module, move them to individual module's subdirectory, example: `/modules/MODULE-NAME/scripts/build` & `/modules/MODULE-NAME/scripts/usage` respectively. 
+- If any additional scripts (apart from **terraform** configs) are required to support the module either to *build* or *use* the module, move them to individual module's subdirectory, example: `/modules/[MODULE-NAME]/scripts/build` & `/modules/[MODULE-NAME]/scripts/usage` respectively. 
 
-- A README.md containing more information about the modules, required IAM permissions and any specific instructions to create the infrastructure.
+- A **README.md** containing more information about the modules, required IAM permissions and any specific instructions to create the infrastructure, etc. Make sure to add atleast below mentioned Headings & Sub Headings:
 
-NOTE: Add below 2 tags in the RAD-Lab module specific README.md for populating structured Variables and Outputs automatically via [tfdoc.py](../tools/tfdoc.py).
+    - HEADING:
+        - `# RAD LAB [MODULE-NAME] Module` : Title of the Module Specific README.
 
-```
-<!-- BEGIN TFDOC -->
-<!-- END TFDOC -->
-```
+    - SUB-HEADINGS:
+        - `## Module Overview` : Description about the module. 
+        - `## GCP Products/Services`: List the GCP products and the sevices used in the respective module development.
+        - `## Reference Architecture Diagram` : Add the GCP architecture diagram under this section. Images for module specific architechture diagrams are stored under [docs/images](./images/) directory.
```

Comment:
We either need a folder and standard name (e.g. `/[MODULE_NAME]/images/architecture.png`) or standardize on the module name being the file name (e.g. `docs/images/[MODULE_NAME]_architecture.png`)

Reply:
Moved GCP architecture diagram under `modules/[MODULE_NAME]/images` with filename as `architecture.png`. Also updated the **CONTRIBUTING.md** for future contributions.